test/Table_Tests/src/Database/SQLite_Spec.enso

```diff
@@ -454,5 +455,6 @@ add_specs suite_builder =
 
 main filter=Nothing =
     suite = Test.build suite_builder->
-        add_specs suite_builder
+        if Platform.os != OS.Mac_OS then
+            add_specs suite_builder
```

Comment:
I don't think this accomplishes what you want to accomplish. This will only skip SQLite tests when you run them as `enso --run test/Table_Tests/src/Database/SQLite_Spec.enso` and not if you run them as `enso --run test/Table_Tests`. I believe that you rather want:
```diff
diff --git a/test/Table_Tests/src/Database/Main.enso b/test/Table_Tests/src/Database/Main.enso
index fc8f6f56f..28dad6384 100644
--- a/test/Table_Tests/src/Database/Main.enso
+++ b/test/Table_Tests/src/Database/Main.enso
@@ -9,7 +9,8 @@ import project.Database.Redshift_Spec
 
 add_specs suite_builder =
     Codegen_Spec.add_specs suite_builder
-    SQLite_Spec.add_specs suite_builder
+    if Platform.os != OS.Mac_OS then
+        SQLite_Spec.add_specs suite_builder
     Postgres_Spec.add_specs suite_builder
     Redshift_Spec.add_specs suite_builder
```

By the way, you can mark a whole group as `pending`.